js/earhart-fellows/EarhartFellowsMergededited.txt

```diff
@@ -25423,8 +25423,8 @@ Bennett, Sponsor, Ph.D., 1956
 WOOD, Carolyn J.
 Graduate Fellowship(s)
 1973-1974
-Washington University, Education, Earhart
-Foundation, Sponsor, Ph.D.
+Washington University, Education,
+EarhartFoundation, Sponsor, Ph.D.

```

Comment:
did you mean to collapse the space here?

Reply:
Nope. Fixing those now